src/redisraft.c

```diff
@@ -1225,7 +1249,7 @@ static int cmdRaftCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int arg
 
         clusterInit(cluster_id);
 
-        char reply[RAFT_DBID_LEN + 256];
+        char reply[RAFT_DBID_LEN + 260];
```

Comment:
did you suspect overflow around here :)

Reply:
I was having compiler complaining, wasn't sure why, but you had same issue.

Reply:
yeah it was false positive